src/main/java/edu/usf/cutr/trackerlib/tracker/batch/BatchUpdateService.java

```diff
@@ -68,6 +69,8 @@ public void onCreate() {
 
         dataManager = new DataManagerImpl(getApplicationContext());
 
+        locationDecimation = new DouglasPeuckerDecimation();
+
```

Comment:
@barbeau I have converted the Douglas-Peucker decimation algorithm from Utility class to interface. In the ```BatchUpdateService``` I start using the new decimation method. If you have any suggestions, I am happy to hear them.